packages/helpers/classes/request.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+
+var _ = require('lodash');
```

Comment:
Generally we're trying to move away from using lodash. Should eval if this can be dropped.